src/server/CMakeLists.txt

```diff
@@ -14,6 +15,7 @@ include_directories(
   ${PROJECT_SOURCE_DIR}/src/include/client
   ${PROJECT_SOURCE_DIR}/src/include/server
   ${PROJECT_SOURCE_DIR}/src/include/wayland
+  ${PIXMAN_INCLUDE_DIRS}
```

Comment:
You shouldn't need this, target_link_libraries over an `IMPORTED_TARGET` adds include dirs automagically.
```suggestion
```

Reply:
Oh well, must be something dumb in the pixman `.pc` file, then.